### DISCUSSION TITLE
No prompt

### DISCUSSION THREAD

**Discussion by rob-jedunn** [cmderdev/cmder#2665] [Q&A] [answered]
I've tried installing cmder several times and keep running into the same issue. When I first run the program, I get feedback that it is generating a couple of files and then nothing else. No prompt at all. When I run it subsequent times, the cmder window opens, and I have a blinking cursor, but no prompt and cannot type anything or ctrl-C, etc. 

What am I missing?

**Comment by daxgames** [comment, marked as answer]
I don't know not really enough info to go on.  Generally you just unzip and run and it works.

Can you post a screenshot of the output you see when you first start it?

Where are you extracting to and running it from?

**Reply by rob-jedunn** [reply]
I'm extracting it to C:\Users\robkre\AppData\Local\cmder.
I'm running it by opening cmder.exe from the same directory.
I'm installing this on a VDI running Windows 10 Enterprise. Not sure if that makes a difference, or possibly something my company managed machine is blocking.

When I first open it, I get this warning:

<img width="537" alt="warning" src="https://user-images.githubusercontent.com/84978888/152668038-4b0ae236-b680-4fbc-9ccb-9beb9655d7df.png">

I then selected "Unblock and continue". Cmder opens and initializes as shown:

<img width="974" alt="init" src="https://user-images.githubusercontent.com/84978888/152668059-e73a47fe-aeba-4956-a96a-0aef311b615d.png">

And then that is it. The cursor blinks but is otherwise unresponsive. You may notice I am on a Mac at the moment. However, I'm running this through Citrix connected to my VDI. I get the same results when running Citrix from a PC.

**Reply by daxgames** [reply]
Everything I saw there is COMPLETELY normal except maybe the chosen location.  Seem kinda deep and hidden.  Why there?

The one place I have personally experienced and heard of others having dufficulties is in the corporate environment.

It is usually due to something installed on the system by the company like overly aggressive Anti Virus or DLP software like Digital Guardian.

I expanded a Cmder install to that location on my system and run it.  My AV pops up and says `bash` might be a virus.  It's not so I tell the AV to create an exception.  Startup completes as expected.  Maybe your AV is just yanking the `bash.exe` into quaranteen and hosing things causing the pause.  Check AV  quarantine if you can.

A component of Cmder 'clink.exe' injects itself into Cmd.exe process and this could be seen as malicious behavior by AV And DLP tools.

Does it work on your home system?

If so you may need to work with your company to determine why itnis happening and how to allow it to work if they are willing.

**Reply by rob-jedunn** [reply]
I installed it in that directory because that is the default directory where everything else I install is going. I think it's because this is a virtual machine (VDI) managed by the corporation. I tried installing it under c:\Program Files but it required me to enter an admin account. And I still had the same result.

It appears you are correct about protection. We use Cylance Protect and when I opened it I see clink listed several times. I tried selecting "unblock" but I'm not sure that actually works for me. I may have to get our security/systems people to look at it.

<img width="648" alt="protect" src="https://user-images.githubusercontent.com/84978888/152700192-b7d864eb-4e67-4cad-9022-0a8c906290ea.png">
